CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -159,6 +166,7 @@ function Disable-DockerDriver {
         Stop-Service docker | Out-Null
         Get-NetNat | Remove-NetNat -Confirm:$false
         Get-ContainerNetwork | Remove-ContainerNetwork -Force
+        Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
```

Comment:
Could you explain this change? Also, why is it almost a duplicate of line 168?

Reply:
These lines should be in reversed order. Could you quickly commit the reversal and let' see what happens? 

For some reason, this line fails in CI, I didn't have time to properly investigate it, so I made this line just run twice as a temporary hack. This should be fixed properly though, either in this or follow up pr.